PowerEditor/installer/nppSetup.nsi

```diff
@@ -43,6 +43,10 @@ OutFile ".\build\npp.${APPVERSION}.Installer.arm64.exe"
 OutFile ".\build\npp.${APPVERSION}.Installer.exe"
 !endif
 
+; Sign both installer and uninstaller
+!finalize        'sign-binaries.bat "%1"' = 0     ; %1 is replaced by the installer exe to be signed.
+!uninstfinalize  'sign-binaries.bat "%1"' = 0     ; %1 is replaced by the uninstaller exe to be signed.

```

Comment:
1. Rename the file sign-binaries.bat to sign-installers.bat.
2. Remove the following lines from [packageAll.bat](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/installer/packageAll.bat#L545) :
``` batch
if %SIGN% == 0 goto NoSignInstaller
%signBinary% !nppInstallerVar!
If ErrorLevel 1 goto End
%signBinary% !nppInstallerVar64!
If ErrorLevel 1 goto End
%signArmBinary% !nppInstallerVarArm64!
If ErrorLevel 1 goto End
:NoSignInstaller
```